build.gradle

```diff
@@ -35,7 +56,24 @@ dependencies {
     testCompile group: 'org.hamcrest', name: 'hamcrest-library', version: '1.3'
 
     compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.14'
+    compile "com.sun.jersey:jersey-json:1.9.1"
     compile 'org.jsoup:jsoup:1.8.1'
     compile 'org.jfrog.buildinfo:build-info-extractor-gradle:3.1.1'
     compile group: 'com.owlike', name: 'genson', version: '0.94'
+    integrationTestCompile 'org.assertj:assertj-core:3.0.0'
+    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.4'
```

Comment:
Do we need `org.apache.httpcomponents` for the main code or only for test? If so it shouldn't it be `testCompile`?

Reply:
yes you are right!